### ISSUE TITLE
Admin UI realm user search loses focus if user federation is enabled

### ISSUE THREAD

**Issue by bjoernhype** [keycloak/keycloak#29488]
### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

admin/ui

### Describe the bug

In Admin UI - realm - users, we can search for a user. In case of user federation enabled in the realm there are some differences, like an additional explaining text, and no initial user list, so we must use the search function. Ok.
Clicking in the search field, I can start typing (as expected), but after the first character, the focus is lost, and I have to click again into the search field, to continue. This only happens with user federation enabled and only after the first character. Disabling an existing user federation hides this problem.

### Version

24.0.4

### Regression

- [X] The issue is a regression

### Expected behavior

User search field can be used as before

### Actual behavior

Focus is lost while typing

### How to Reproduce?

Realm with enabled user federation.
Open the users page and try to search for a user

### Anything else?

With Keycloak 20.x, this worked without problems. I'm unsure which exact version did introduce this. We upgraded to 24.0.2 and than 24.0.4, where it appears in both.

**Comment by ssilvert** [comment]
This is fixed in the nightly build.

Also, this issue is a duplicate of https://github.com/keycloak/keycloak/issues/29153

**Comment by bjoernhype** [comment]
Thanks :)
and sorry I didn't find the duplicate
